src/pages/mapping/MappingSetting.tsx

```
@@ -0,0 +1,374 @@
+                                                            actionId: action.id,
+                                                        };
+                                                        return m;
+                                                    }
+                                                    return prev;
+                                                })
+                                            }
+                                        >
+                                            {plugin.manifest.name} : {action.name}
+                                        </Button>
+                                    );
+                                });
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)